go.mod

```diff
@@ -12,3 +12,5 @@ require (
 	k8s.io/code-generator v0.19.2
 	k8s.io/klog/v2 v2.3.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/stlaz/api v0.0.0-20201112122008-c729dd5d5b82
```

Comment:
This is wrong, was this already merged to openshift/api?

Reply:
it hasn't, I completely forgot about this, had it locally for some time. Glad you caught it.